From a879fc32d58625bd9361219c1285ab6e5f012467 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Fri, 2 Nov 2018 14:29:53 +0100
Subject: [PATCH] monitor/pcap.c: add missing sys/stat.h include
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Fixes:

    CC       monitor/pcap.o
  monitor/pcap.c: In function ‘pcap_create’:
  monitor/pcap.c:121:6: error: ‘S_IRUSR’ undeclared (first use in this function)
        S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
        ^
  monitor/pcap.c:121:6: note: each undeclared identifier is reported only once for each function it appears in
  monitor/pcap.c:121:16: error: ‘S_IWUSR’ undeclared (first use in this function)
        S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
                  ^
  monitor/pcap.c:121:26: error: ‘S_IRGRP’ undeclared (first use in this function)
        S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
                            ^
  monitor/pcap.c:121:36: error: ‘S_IROTH’ undeclared (first use in this function)
        S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
                                      ^

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 monitor/pcap.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/monitor/pcap.c b/monitor/pcap.c
index 54ad9f5..785aefa 100644
--- a/monitor/pcap.c
+++ b/monitor/pcap.c
@@ -29,6 +29,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <stdlib.h>
+#include <sys/stat.h>
 #include <sys/uio.h>
 #include <sys/types.h>
 #include <ell/ell.h>
-- 
2.19.1

